src/en/proposals/fairlysadpanda-revs-uplink.md

```diff
@@ -0,0 +1,160 @@
+
+During rounds, it's the fancy way of sending communications. It's like when a captain in a Star Trek episode needs to stand somewhere on the bridge to send a message to the crew to abandon ship or something.
+
+## The Terminal Is Robust
+
+The terminal is really hardy. It's an indepentent computer structure which is indepentently powered and has a high amount of structure damage resistance. It can't be broken into with tools or unanchored with tools. Essentially, it's as hardy as CentComm can reasonably make it. It can still be nuked, eaten by a singuloose, and so on.
+
+## (It is not infinitely robust)
+
+If the terminal is destroyed (except by nuke) then CentComm will send an automated annoyed message to the crew and send a replacement computer board to the automated trade station. This is absolutely a way that the Revolution could game the system, but considering that they'd need to do something very drastic to do so...
+
+## Gameplay Role
+
+The Terminal if de-linked from CentComm or destroyed will cause all security systems on the station to offline, meaning all IDs become AA.
+
+The Terminal can be used to send communications and set alert levels just like a Communications Computer. Its robust nature means it's also a reliable place to call the shuttle from.
+
+The Terminal monitors the existence of the Head ID cards. Anyone can look at the terminal and know how many of the IDs are still "in play" - i.e. not destroyed or unreasonably far from the space station for recovery.
+
+In Revolutionary rounds, the Terminal can be exploited by a Head Revolutionary to initiate an unlinking sequence. This unlinking sequence works as follows:
+
+1. The Head Revolutionary gets at least one Head ID card.
+2. The Head Revolutionary breaks into the room with the terminal in it (probably the bridge).
+3. The Head Revolutionary\* inserts the ID card into the terminal.
+4. The Head Revolutionary waits through the timer.
+5. If the Head Revolutionary has any more Head ID cards, they can insert them to decrease the length of the timer.
+
+This timer blares out a warning that the station is being unlinked when the timer starts, with the time remaining. If the time remaining is modified (by inserting more ID cards), this is also announced.
+
+Any crew member can stop the countdown by interacting with the terminal and clicking a big obvious red flashing button on it. This stops the timer\*\*.
+
+If the timer reaches zero, the Revolutionaries win and a round restart is called. The Head Revolutionaries exploit the console to disable all security ~~ready for the Syndicate to show up~~ to free the crew from their corporate overlords and install themselves as "Consuls" of the station\*\*\*. Remaining living department heads and security officers are renamed "Enemies of the Revolution" and are ultra-valid, although given that the game is in end-of-round the murder has probably already started.
```

Comment:
This design doc highlights that one of the main issues of revolutionaries is murderboning, but doesn't this climax inevitably result in a large chunk of players getting round removed from said murderboning?

Security will have to use lethals against a revolution this far in, and obviously revs will fight back, most likely resulting in a very large ammount of casualties. If heads win, everyone deconverts which allows the station to treat the wounded, but if a headrev escapes/hides, wouldn't the heads be kind of forced to leave the revs to rot?

Reply:
Murderboning: Yes, but it's the finale of the round. If the revs lose the bridge then they've probably lost and are on the run. It's more fun to die at an interesting finale point and watch the mop-up from ghosting than dying at the half-way point.

Leaving folks to rot: Under the MindShield meta: no, as deconversion of a corpse means reviving them and sticking a shield in their brain. Other mechanics will work differently but I'm hoping whatever is agreed on gives security an AOE way of dealing with the converted.